content/ember/v6/deprecate-array-proxy.md

```diff
@@ -0,0 +1,208 @@
+---
+title: Deprecation of ArrayProxy
+until: 7.0.0
+since: 6.5.0
+---
+
+`ArrayProxy` is deprecated. In modern Ember, you can replace `ArrayProxy` with native JavaScript classes and proxies. The best replacement depends on how you were using `ArrayProxy`. Some example use cases are shown below.
+
+### Swapping Content
```

Comment:
These guides show a way to achieve something technically the same as what people had before -- which is great -- but I also think we should guide users into what the modern Ember pattern would do instead. In this case, I believe that would mostly be tracked-built-ins.

Reply:
My general point is that we should point people to "the Ember way" going forwards, not just a strict technical replacement -- why did they want a swappable container? Do you still need that with @tracked? (and applying to the other guides not just this first one that I commented on).